src/sentry/models/appconnectbuilds.py

```diff
@@ -0,0 +1,54 @@
+"""Database models to keep track of the App Store Connect builds for a project.
+
+If a project enables the App Store Connect source to download dSYMs directly from Apple we
+need to keep track of which builds have already been downloaded.
+"""
+
+from django.db import models
+
+from sentry.db.models import FlexibleForeignKey, Model
+
+
+class AppConnectBuild(Model):
```

Comment:
It's probably worth adding this to https://github.com/getsentry/sentry/blob/38abee742ce6a746f5687f02014aded786d99344/src/sentry/deletions/defaults/project.py#L17, since I could imagine we might have a lot of these?

Reply:
makes sense, thanks!